src/main/java/seedu/address/model/event/Event.java

```diff
@@ -30,6 +36,17 @@ public Event(String eventName, LocalDate date, Set<Person> attendees) {
         requireAllNonNull(eventName, date, attendees);
         this.eventName = eventName;
         this.date = date;
+
+        // buffer data to show UI changes, since adding attendees are not yet implemented.
```

Comment:
I think you can use `SampleDataUtil.getSamplePersons()` here

Reply:
Thanks for pointing it out, I've changed it in the new pr.